_rules/active-focus-shortcut-1e9941.md

```diff
@@ -21,15 +21,15 @@ acknowledgements:
 
 ## Applicability
 
-The rule applies to any [HTML document][] with at least one [keyboard shortcut][] that requires pressing only [printable character][] keys to trigger an action on a [user interface component][].
+The rule applies to any [keyboard shortcut][] that requires pressing only [printable character][] keys to trigger an [event][] in an [HTML element][] that can receive [focus][] within a [HTML document][].

```

Comment:
I am not truly getting this I think. How can an "event" trigger "in an HTML element"? Could you give an example of an event that triggers in an HTML element as oppose to that event triggering outside the HTML element?